src/engine.ts

```diff
@@ -75,6 +85,25 @@ export class ChronerEngine extends Engine<never, ChronerTask> {
     }
   }
 
+  acquireEffects(task: ChronerTask): void {
+    if (task.do instanceof Location) {
+      if (task.effects === undefined) task.effects = [];
+      task.effects = task.effects.concat(
```

Comment:
Since this changes the base task.effects, it will add repeats of these effects to the task every time the same task is executed.

Reply:
I think it might be worth attaching this to the Quest instead of to individual tasks. And to avoid accumulating extras, we can convert from array to Set to array.

Alternately, we might just want effects to be a Set in some later version of grimoire.